### ISSUE TITLE
D1 Mini powers up in REST mode, no way to configure AP/iSpindel

### ISSUE THREAD

**Issue by PJRanger** [universam1/iSpindel#280]
I love this project and have 2 iSpindels built however I cannot get the ESP8266 into AP mode to access the configuration page.  It powers on in the REST mode, the AP is visible on wifi but cannot connect to the config page at 192.168.4.1   Holding the RESET button for 5 seconds does nothing other than blink the light.  Please help.

**Comment by ErikdBr** [comment]
You have to connect to the ap first and then go to the ip adress.

**Comment by PJRanger** [comment]
Erik, this is the issue, when I load the firmware to the ESP8266 the AP will allow a connection however will not accept an IP connection at the address 192.168.4.1 (config HTTP page does not display). In the documentation it states with the iSpindel Firmware loaded the ESP8266 will boot in the rest mode and has to be set to AP mode by the reset button, however the reset button has no effect and does not go into AP mode (LED flashing at 1 second). I can access the AP but cannot enter configuration mode to proceed with any calibration, ubidots configuration or connection to my wifi network.

**Comment by universam1** [comment]
Post the condole log output here. Anything else is wild guessing

**Comment by PJRanger** [comment]
These are screen shots of my NodeMCU loading of the firmware and a text file of the log.

**Comment by PJRanger** [comment]
Here are all the screenshots and the log from NodeMCU, is there another log file I should be looking for and if so how do I get to it?

![Screenshot 2](https://user-images.githubusercontent.com/44042347/59151502-eeeecc00-89f9-11e9-9c8d-154d9ec117fa.png)
![Screenshot 3](https://user-images.githubusercontent.com/44042347/59151504-f3b38000-89f9-11e9-8bd6-e12ad431ecaa.png)
![Screenshot 6](https://user-images.githubusercontent.com/44042347/59151506-f6ae7080-89f9-11e9-96e6-31eedbe6968d.png)
![Screenshot 7](https://user-images.githubusercontent.com/44042347/59151507-f910ca80-89f9-11e9-96bf-da40910a8eb1.png)
![Screenshot 1](https://user-images.githubusercontent.com/44042347/59151509-fb732480-89f9-11e9-9717-97b1c789313a.png)
[Log from NodeMCU.txt](https://github.com/universam1/iSpindel/files/3268864/Log.from.NodeMCU.txt)
![Photo Jun 08, 14 10 39](https://user-images.githubusercontent.com/44042347/59151549-8fdd8700-89fa-11e9-8b95-883d6163b985.jpg)

**Comment by ErikdBr** [comment]
After the initial flash you have to flash it with the Ispindel firmware. Then you will see "ispindel" in your wifi list and not "ESP_A3B626".

**Comment by ErikdBr** [comment]
Firmware you will find here: https://github.com/universam1/iSpindel/releases

**Comment by PJRanger** [comment]
I actually figured it out.  The flashing procedure using NodeMCU should mention to remove or deselect the NODEMCU flash information and just select the iSpindel firmware.  Once I did that my iSpindel worked as advertised.  I now have two communicating iSpindels using FERMENTRACK and will also work to connect them to CraftBeerPi.  Thank you for the assistance.  If needed, I do know some German and could help in writing documentation in English.
